### ISSUE TITLE
Documentation should show proper use of throttle 

### ISSUE THREAD

**Issue by ArbitRandomUser** [FluxML/Flux.jl#1399]
https://fluxml.ai/Flux.jl/stable/training/training/
documentation shows 

```
Flux.train!(objective, ps, data, opt,
            cb = throttle(evalcb, 5))
```

however shouldnt throttle be assigned first to some variable `f = throttle(evalcb,5)`
and then `train!` called  with `cb=f` ?

because the way it is shown in the docs the function seems to be getting called every epoch irrespective of the time passed to throttle

**Comment by ToucheSir** [comment]
Ah, nice catch. Do you mind putting in a quick docs PR for this?

**Comment by CarloLucibello** [comment]
fixed in #1400
